go-controller/pkg/cni/types/types.go

```diff
@@ -18,7 +18,11 @@ type NetConf struct {
 	MTU int `json:"mtu,omitempty"`
 	// comma-seperated subnet cidr
 	// for secondary layer3 network, eg. 10.128.0.0/14/23
+	// for layer2 network, eg. 10.1.130.0/24
 	Subnets string `json:"subnets,omitempty"`
+	// list of IPs, expressed with prefix length, to be excluded from being allocated for Pod
+	// valid for localnet or layer 2 network topology
+	ExcludeCIDRs []string `json:"excludeCIDRs,omitempty"`
```

Comment:
I agree. As I mentioned in the meeting, the way to use comma separated string for subnets is only because it is consistent with how it is defined for cidr for default network in ovn configmap.

perhaps we can allow either comma based string or array for "subnets"?

Reply:
Well, out of all the possible options, certainly a comma separated string is the least json of all.

I don't know why we would need to be aligned to the way it is done in the OVN config map. If it were me it would be that the one that I would change to add support a yaml array.

In any case, here we already decided for the coma separated string. So I would stick with that as well for ExcludeCIDRs for now.